libdevcore/OverlayDB.cpp

```diff
@@ -135,18 +130,24 @@ void OverlayDB::kill(h256 const& _h)
 	{
 		std::string ret;
 		if (m_db)
-			m_db->Get(m_readOptions, ldb::Slice((char const*)_h.data(), 32), &ret);
-		// No point node ref decreasing for EmptyTrie since we never bother incrementing it in the first place for
-		// empty storage tries.
-		if (ret.empty() && _h != EmptyTrie)
-			cnote << "Decreasing DB node ref count below zero with no DB node. Probably have a corrupt Trie." << _h;
-
-		// TODO: for 1.1: ref-counted triedb.
+		{
+			try
+			{
+				ret = m_db->lookup(db::Slice(reinterpret_cast<char const*>(_h.data()), 32));
```

Comment:
I think can just use `m_db->exists(...)` here instead of `lookup` in `try-catch`